drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c

```diff
@@ -778,6 +859,9 @@ static int bcmf_txavail(FAR struct net_driver_s *dev)
 {
   FAR struct bcmf_dev_s *priv = (FAR struct bcmf_dev_s *)dev->d_private;
 
+#ifdef CONFIG_IEEE80211_BROADCOM_LOWPOWER
+  bcmf_lowpower_poll(priv);
```

Comment:
put to work thread?

Reply:
tx work thread will try again after receive data, it is more appropriate to poll low power in txavailable

https://github.com/apache/incubator-nuttx/blob/master/drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c#L226-L232